commerce-add-ons/payment-providers/quickpay/configuring-quickpay.md

```diff
@@ -15,3 +15,34 @@ If you haven't done so yet, head over to the [QuickPay portal](https://manage.qu
 Or [login to the QuickPay portal](https://manage.quickpay.net/) if you already have an account.
 
 ![The login form to the QuickPay portal.](../media/quickpay/quickpay\_signin.png)
+
+## Step 2: API Keys
```

Comment:
As you start with step 2, can you please update the previous headline with "Step 1" to make it consistent where it starts?

Reply:
I've added Step 1 to the first headline, to align with the other articles :)